skills/csharp/experimental/bingsearchskill/Dialogs/SearchDialog.cs

```diff
@@ -58,11 +58,12 @@ private async Task<DialogTurnResult> PromptForQuestion(WaterfallStepContext step
 
             var state = await _stateAccessor.GetAsync(stepContext.Context);
 
-            // if (string.IsNullOrWhiteSpace(state.SearchEntityName))
-            // {
-            //    var prompt = ResponseManager.GetResponse(SearchResponses.AskEntityPrompt);
-            //    return await stepContext.PromptAsync(DialogIds.NamePrompt, new PromptOptions { Prompt = prompt });
-            // }
+            if (string.IsNullOrWhiteSpace(state.SearchEntityName))
+            {
+                var prompt = TemplateManager.GenerateActivity(SearchResponses.AskEntityPrompt);
+                return await stepContext.PromptAsync(DialogIds.NamePrompt, new PromptOptions { Prompt = prompt });
+            }
+
```

Comment:
should still be commented? since stepContext.result is not used in the following step

Reply:
no. it prompts user to input and get the result in next step. The code is commented because lg issue but forgot to uncomment after lg has been fixed..

Reply:
oh.. I see. it uses stepContext.Context.Activity.Text instead of stepContext.result.
Would you mind change it? Since it is quite unusual..

Reply:
yes... I will merge it first then fix it in next change because it is blocking other PRs